src/Bridge/Doctrine/Orm/Util/QueryBuilderHelper.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Doctrine\Orm\Util;
+
+use Doctrine\ORM\Query\Expr\Join;
+use Doctrine\ORM\QueryBuilder;
+
+/**
+ * @author Vincent Chalamon <vincent@les-tilleuls.coop>
+ *
+ * @internal
+ */
+final class QueryBuilderHelper
```

Comment:
Why for?

Reply:
Because he likes private constructors :P.

Nah just that it's a convention when classes have no instances (ie are never called with `new` because they only have statics).